roles/kubernetes/master/templates/kubeadm-config.v1alpha2.yaml.j2

```diff
@@ -81,6 +75,13 @@ apiServerExtraArgs:
   runtime-config: {{ kube_api_runtime_config | join(',') }}
 {% endif %}
   allow-privileged: "true"
+{% if kubernetes_audit %}
```

Comment:
delete futrue audit.

Reply:
I don't see your point. Can you explain?

Reply:
Sorry, delete  kubeadm Auditing featureGates.

Reply:
@riverzhang why deleting the feature gate? Don't we still need it to activate auditing?